flow-tests/test-resources/src/main/java/com/vaadin/flow/uitest/ui/dependencies/TestVersion.java

```diff
@@ -2,7 +2,7 @@
 
 public class TestVersion {
 
-    public static final String VAADIN = "22.0.0-alpha9";
+    public static final String VAADIN = "23.0.4";
```

Comment:
```suggestion
    public static final String VAADIN = "23.0.5";
```
As now we are getting mixed 23.0.4 and 23.0.5 in the package lock file.

Reply:
We can change that but it will break the same way when 23.0.6 is out

Reply:
Should we then just have an empty package.json files that get populated instead to not get snyk pr's for internal testing things?

Reply:
Let's upgrade to 23.0.5 and see how it goes